docs/configuring_tor.md

```diff
@@ -1,54 +1,49 @@
 # Table of Contents
-1. [Overview](#Overview)
-2. [Outbound Connections Only](#outbound-connections-only)
+1. [Overview](#overview)
+2. [Getting Started](#getting-started)
 3. [Tor Stream Isolation](#tor-stream-isolation)
+4. [Listening for Inbound Connections](#listening-for-inbound-connections)
+	1. [v2 Onion Services](#v2-onion-services)
+	2. [v3 Onion Services](#v3-onion-services)
 
-## 1. Overview
+## Overview
 
-`lnd` currently has _partial_ support for using Lightning over
+`lnd` currently has complete support for using Lightning over
 [Tor](https://www.torproject.org/). Usage of Lightning over Tor is valuable as
 routing nodes no longer need to potentially expose their location via their
 advertised IP address. Additionally, leaf nodes can also protect their location
-by using Tor for anonymous networking to establish connections.  
+by using Tor for anonymous networking to establish connections.
 
-At the time of the writing of this documentation, `lnd` only supports usage of
-Tor for establishing _outbound_ connections. In the near future, support for
-full [Onion Service](https://www.torproject.org/docs/onion-services.html.en)
-usage will be added as well. Support for both `v2` and `v3` onion services are
-planned. With widespread usage of Onion Services within the network, concerns
-about the difficulty of proper NAT traversal are alleviated, as usage of Onion
-Services allows nodes to accept inbound connections even if they're behind a
-NAT.
+With widespread usage of Onion Services within the network, concerns about the
+difficulty of proper NAT traversal are alleviated, as usage of Onion Services
+allows nodes to accept inbound connections even if they're behind a NAT.
+
+At the time of writing this documentation, `lnd` supports both types of onion
+services: v2 and v3. However, only v2 onion services can automatically be
+created and set up by `lnd` until Tor Control support for v3 onion services is
+implemented in the stable release of the Tor daemon. v3 onion services can be
+used as long as they are set up manually. We'll cover the steps on how to do
+these things below.
 
 Before following the remainder of this documentation, you should ensure that
 you already have Tor installed locally. Official instructions to install the
 latest release of Tor can be found
 [here](https://www.torproject.org/docs/tor-doc-unix.html.en).
 
 **NOTE**: This documentation covers how to ensure that `lnd`'s _Lightning
-protocol traffic_ is tunnled over Tor. Users will need to take care that if
-they're running using a Bitcoin full-node, then that is also configured to
-proxy all trafic over Tor. If using the `neutrino` backend for `lnd`, then it
-will automatically also default to Tor usage if active within `lnd`.
-
-
-## 2. Outbound Connections Only
+protocol traffic_ is tunneled over Tor. Users must ensure that when also running
+a backend node, that it is also proxying all traffic over Tor. If using the
```

Comment:
I think we should keep the "Bitcoin full-node" jargon here, as "backend node" might not be known to as many.

Reply:
Fixed.